### PR TITLE
On boarding in layers not working

### LINKED ISSUE #12192: SMOKE_OB07 On boarding not working in layers
### Context
The on boarding of data and style inside layers is only showing step 1, when you click on "Next" it closes the on boarding.

### Steps to Reproduce
1. Create a new layer with a dataset (or without dataset and add a geometry)
2. Enter in the layer, it should open the on boarding of the style tab
3. Click "Take the tour"
4. Click "Next"

### Current Result
It closes the on boarding

### Expected result
It shows step 2 of the on boarding

### Browser and version
Google Chrome Version 58.0.3029.110 (64-bit)

### PR DESCRIPTION
**Pull request by rubenmoya**
This PR fixes #12192

On boardings were not working in the  **Data** and **Style** sections  when using the **Next** button.
 
stoppropagation

### Acceptance
- [ ] Data on boarding works correctly using the mouse
- [ ] Style on boarding works correctly using the mouse

### CODE REVIEW

**Comment by xavijam** [comment]
Ouch, @nobuti and @rjimenezda, I've already reviewed it, but it is all yours, sorry :joy:.

File: lib/assets/core/test/spec/cartodb3/components/onboardings/generic/generic-onboarding-view.spec.js [hunk 1]
```diff
@@ -139,6 +139,17 @@ describe('components/onboardings/generic/generic-onboarding-view', function () {
     });
   });
 
+  describe('_onClickNext', function () {
+    it('should stop event propagation', function () {
+      var stopPropagationCalled = false;
+      var event = {
+        stopPropagation: function () { stopPropagationCalled = true; }
```
Comment (rjimenezda) [inline lib/assets/core/test/spec/cartodb3/components/onboardings/generic/generic-onboarding-view.spec.js]: This is 👌, but why not use spyOn to get a mock and check hasBeenCalled?

Comment (rubenmoya) [inline lib/assets/core/test/spec/cartodb3/components/onboardings/generic/generic-onboarding-view.spec.js]: To be honest, I saw it in the _onKeyDown tests, and copied it, I'll try to use spyOn and hasBeenCalled and change it in both tests if it works.

**Comment by xavijam** [comment]
@rjimenezda, ready to be moved to acceptance?

**Pull request merged**